api/openapi-spec/v3/api__v1_openapi.json

```diff
@@ -3737,6 +3737,16 @@
         },
         "type": "object"
       },
+      "io.k8s.api.core.v1.NodeFeatures": {
+        "description": "NodeFeatures describes the set of features implemented by the CRI implementation. The features contained in the NodeFeatures should depend on only cri implementation, be independent of runtime handlers, (i.e. it should not require inspecting any information from the OCI runtime-spec's features).",
```

Comment:
```suggestion
        "description": "NodeFeatures describes the set of features implemented by the CRI implementation. The features contained in the NodeFeatures should depend only on the cri implementation independent of runtime handlers.",
```

Reply:
fixed in 
- K8s API: 5d75660dc11ff443ebab2551aed8e56a54cc218d
- CRI: 38e4c6b01666128216290e1579a8f2d408a60e66